Snakefile

```diff
@@ -27,7 +26,8 @@ def get_dataset_dependencies(datasets, label):
     all_files = dataset["node_files"] + dataset["edge_files"] + dataset["other_files"]
     # Add the relative file path and config file
     all_files = [os.path.join(dataset["data_dir"], data_file) for data_file in all_files]
-    return all_files + [config_file]
+    # TODO Need to restore dependency on the config file
```

Comment:
This is complex and related to the changes in #41.  We will work on it in a separate pull request or #41.

Reply:
Ok, I've added these edits.